tests/chainerx_tests/unit_tests/routines_tests/test_activation.py

```diff
@@ -75,6 +75,37 @@ def forward_xp(self, inputs, xp):
 ]
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': _in_out_dtypes_math_functions,
+        'input': [-2, 2],
+        'contiguous': [None, 'C'],
+    })
+    # Special values
+    + chainer.testing.product({
+        'shape': [(2, 3)],
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
+        'input': [0, float('inf'), -float('inf'), float('nan')],
+        'skip_backward_test': [True],
+        'skip_double_backward_test': [True],
+    })
+))
+class TestElu(UnaryMathTestBase, op_utils.NumpyOpTest):
+
+    alpha = 2.0
```

Comment:
Please parameterize `alpha`, including a negative value, zero and "unspecified".